ethcore/sync/src/chain/propagator.rs

```diff
@@ -270,7 +297,7 @@ impl SyncPropagator {
                 .chain(&to_send)
                 .cloned()
                 .collect();
-            send_packet(io, peer_id, to_send.len(), packet.out());
+            send_packet(io, peer_id, is_hashes, to_send.len(), packet.out());
```

Comment:
Hmm, `last_sent_transactions` and `stats.propagated` will now think that the peers knows about these transactions already, but in fact it only knows about the hashes. I supposed it's good enough though, but might mess up some statistics.